static/app/components/confirm.tsx

```diff
@@ -38,6 +38,14 @@ type Props = {
    * Callback when user confirms
    */
   onConfirm?: () => void;
+  /**
+   * Custom function to render the confirm button
+   */
+  renderConfirmButton?: (closeModal: () => void, defaultOnClick: () => void) => void;
```

Comment:
@Neo-Zhixing these functions should return something like a `React.Node` instead of void

Reply:
Also, I think the standard usually makes the render function something like:
```
(props: {closeModal: () => void, defaultOnClick: () => void})
```
so it's more like passing named props to a component instead of ordered arguments